flink-python/src/test/java/org/apache/flink/table/runtime/operators/python/aggregate/arrow/stream/StreamArrowPythonProcTimeBoundedRangeOperatorTest.java

```diff
@@ -109,38 +114,53 @@ public AbstractArrowPythonAggregateFunctionOperator getTestOperator(
             RowType outputType,
             int[] groupingSet,
             int[] udafInputOffsets) {
+        RowType userDefinedFunctionInputType =
+                (RowType) Projection.of(udafInputOffsets).project(inputType);
+        RowType userDefinedFunctionOutputType =
+                new RowType(
+                        outputType
+                                .getFields()
+                                .subList(inputType.getFieldCount(), outputType.getFieldCount()));
+        GeneratedProjection generatedProjection =
```

Comment:
nit: inline this into the parameter for consistency? currently the PR does some with a local variable and most of them inline as a parameter.